packages/components/src/internal/components/domainproperties/dataclasses/DataClassDesigner.tsx

```diff
@@ -59,7 +60,7 @@ class DataClassDesignerImpl extends PureComponent<Props & InjectedBaseDomainDesi
     }
 
     componentDidMount = async (): Promise<void> => {
-        if (this.state.model.isNew) {
+        if (this.state.model.isNew && isSampleManagerEnabled()) {
```

Comment:
I felt the function `isSampleManagerEnabled()` to be slightly confusingly named, as I would have expected it to return a bool based on whether the module is enabled in a folder, but actually relates to whether the module is on the server context's module context and works to determine whether the module is built. I would be down to change it to something like `isSampleManager`-Built/Existent/Present, but also happy to leave as-is.